package.json

```diff
@@ -34,13 +34,14 @@
     "recompose": "^0.30.0"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "BROWSER=firefox react-scripts start",
```

Comment:
We shouldn't save this in `npm start` as not everyone uses firefox, those who need it for testing can set it either with .env or manually in their terminal